accounts/selenium_test.py

```diff
@@ -0,0 +1,38 @@
+import unittest
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
+from selenium.common.exceptions import NoSuchElementException
+USERNAME = 'openstax_cms_tester'
+PASSWORD = 'openstax_cms_tester'
+
+class Integration(unittest.TestCase):
```

Comment:
This test will need to change to test login from /admin - the template in the PR needs to be removed.